content/docs/gke/authentication.md

```diff
@@ -0,0 +1,175 @@
++++
+title = "Authenticating Kubeflow to GCP"
+description = "Troubleshooting guide for authentication and authorization to GCP"
+weight = 4
++++
+
+
+# Authenticating gcloud
+
+[The `gcloud` tool](https://cloud.google.com/sdk/gcloud/) is used to interact with Google Cloud Platform (GCP) over the command line. 
+It can be used to [set up Google Kubernetes Engine (GKE) clusters](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create), 
+and interact with other Google services.
+
+### Logging in
+
+There are two ways to authenticate the gcloud command:
+
+
+- A **service account** is an account set up within your GCP project. Authentication is handled by 
+[downloading a `.json` key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) 
+associated with the account, and registering it with gcloud using 
+[`gcloud auth activate-service-account`](https://cloud.google.com/sdk/gcloud/reference/auth/activate-service-account).
+This is the recommended way to authenticate with GCP.
+
+- A **user account** is a Google (typically Gmail) account that users can use to authenticate.
+You can register a user account using [`gcloud auth login`](https://cloud.google.com/sdk/gcloud/reference/auth/login), 
+which brings up a browser window to start the familiar Google authentication flow.
+
+More information can be found in the [GCP docs](https://cloud.google.com/sdk/docs/authorizing).
+
+### Listing Active Accounts
+
+You can run the following commend to verify you are authenticating with the expected account. 
+Your active account will be denoted with an asterisk.
```

Comment:
We avoid future tense and passive voice in the docs. See the style guide for reasons:
https://www.kubeflow.org/docs/about/style-guide/

So, change this:

`Your active account will be denoted with an asterisk.`

To this:

`In the output of the command, an asterisk denotes your active account.`